broad-wb-malkov.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "terra",
+  "intent": "terra infrastructure",
+  "production": false,
+  "stability": "volatile",
+  "environments": {
+    "default": {
+      "name": "dev"
+    }
+  },
+  "projects": {
+    "default": {
+      "name": "broad-wb-malkov",
+      "provider_sa_path": "secret/dsde/terra/envs/common/terra-env-owner-service-account.json"
+    },
+    "state_bucket": {
+      "name": "broad-dsp-techops",
+      "provider_sa_path": "secret/dsp/service-accts/dsp-techops/terraform-state-service-account.json"
+    }
+  },
+  "regions": {
+    "default": {
+      "name": "us-central1",
+      "zone_name": "us-central1-a"
+    }
+  },
+  "dns": {
+    "default": {
+      "name": "dsp-techops",
+      "zone": "dsp-techops.broadinstitute.org"
+    }
+  },
+  "terraform_state": {
+    "default": {
+      "state_path_prefix":"dsp-techops",
```

Comment:
while this won't work - I think it is blurring things.  I think we should put all individual state in object path that has nothing to do with dsp-techops.  When I did my first env the path prefix was "workbench"  and then I added a terra env identifier.  I think we should go ahead and use the "workbench" object path and decide what the next level should be.